src/modules/dashboard/components/CreatePaymentDialog/CreatePaymentDialogForm.tsx

```diff
@@ -242,18 +244,28 @@ const CreatePaymentDialogForm = ({
     fromDomainRoles,
     ColonyRole.Administration,
   );
-  const userHasPermission =
-    userHasFundingPermission && userHasAdministrationPermission;
+  const hasRoles = userHasFundingPermission && userHasAdministrationPermission;
   const requiredRoles: ColonyRole[] = [
     ColonyRole.Funding,
     ColonyRole.Administration,
   ];
 
-  const canMakePayment =
+  const [userHasPermission, onlyForceAction] = useDialogActionPermissions(
+    colony.colonyAddress,
+    hasRoles,
+    isVotingExtensionEnabled,
+    values.forceAction,
+  );
+
+  const isPaymentExtensionInstalled =
```

Comment:
We can use useEnabledExtensions hook which have isOneTxPaymentExtensionEnabled flag. This one is not enough either way as we need to make sure that this extension is enabled

Reply:
this was already the code in both instances but a very good catch!